foodplanner/lib/models/class.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+class Class {
+  final int classId;
+  final String className;
+
+  const Class({
+    required this.classId,
+    required this.className,
+  });
+
+  factory Class.fromJson(Map<String, dynamic> json) {
+    return switch (json) {
+      {
+        'classId': int id,
+        'className': String className,
+      } =>
+        Class(
+          classId: id,
+          className: className,
+        ),
+      _ => throw const FormatException('Klasse kunne ikke findes.'),
+    };
+  }
+
+  static List<Class> fromJsonList(List<dynamic> jsonList) {
+    return jsonList
+        .map((json) => Class.fromJson(json as Map<String, dynamic>))
+        .toList();
+  }
+}
+
+Future<List<Class>> fetchAllClasses() async {
+  final response =
+      await http.get(Uri.parse('http://127.0.0.1:8080/api/Classrooms/GetAll'));
```

Comment:
Vi har lavet en Base.Url i en config file for services, som kan bruges til alle fetches. Ligger i vores PR secure storage. :)